assessmentModel/src/commonMain/kotlin/serialization/Nodes.kt

```diff
@@ -171,11 +201,53 @@ data class SectionObject(override val identifier: String,
 
 @Serializable
 @SerialName("instruction")
-data class InstructionStepObject(override val identifier: String,
-                                 override val resultIdentifier: String? = null,
-                                 @SerialName("image")
-                                 override var imageInfo: ImageInfo? = null,
-                                 override var fullInstructionsOnly: Boolean = false) : StepObject(), InstructionStep
+data class InstructionStepObject(
+    override val identifier: String,
+     override val resultIdentifier: String? = null,
+     @SerialName("image")
+     override var imageInfo: ImageInfo? = null,
+     override var fullInstructionsOnly: Boolean = false
+) : StepObject(), InstructionStep
+
+@Serializable
+@SerialName("overview")
+data class OverviewStepObject(
+    override val identifier: String,
+    override val resultIdentifier: String? = null,
+    override var imageInfo: ImageInfo? = null,
+    override var icons: List<ImageInfoObject>? = null,
+    override var permissions: List<PermissionInfoObject>? = null
+) : StepObject(), OverviewStep {
+    override var learnMore: ButtonActionInfo?
+        get() = buttonMap[ButtonAction.Navigation.LearnMore]
+        set(value) = setButton(ButtonAction.Navigation.LearnMore, value)
+}
+
+@Serializable
+data class PermissionInfoObject(
+    override val permissionType: PermissionType,
+    override val optional: Boolean = false,
+    override val requiresBackground: Boolean = false,
+    override val reason: String? = null,
+    override val restrictedMessage: String? = null,
+    override val deniedMessage: String? = null
+) : PermissionInfo
+
+@Serializable
+data class ImageInfoObject(
```

Comment:
Should this be called IconInfoObject since it is specific to the icons on overview steps?  Otherwise maybe it should be in the Image.kt file with FetchableImage.

Reply:
Ock, yes. Not sure why I'd changed the name. Its icon info in SageResearch.